lib/tasks/backup.rake

```diff
@@ -45,8 +41,6 @@ namespace :db do
     case config['adapter']
     when 'postgresql'
       postgres_dump(backup_name, config, tables)
-    when 'sqlite3'
-      sqlite_dump(backup_name, config)
     else
       puts 'Your database is not supported by Foreman.' and exit(1)
     end
```

Comment:
It feels like we can do this check at line 23 and error out early, but I'm ok with keeping this patch minimal.